libs/shared/ui/src/lib/components/modals/modal-crud/modal-crud.tsx

```diff
@@ -81,6 +83,7 @@ export function ModalCrud(props: ModalCrudProps) {
       <form className="mt-6" onSubmit={onSubmit}>
         {children}
         <div className="mt-6 flex justify-end gap-3">
+          {bottomButtons}
```

Comment:
I added this new props because we've more and more button at the left of this modal, let me know if it's fine!